packages/ui-components/src/accordian/Accordian.tsx

```diff
@@ -66,43 +66,44 @@ const AccordianContainer: FC<AccordianProps> = ({
   renderBody,
   isActive,
   setActiveIndex,
-}) => {
-  return (
-    <div className="accordion">
-      {items.map((item, index) => {
-        return (
-          <AccordianItem
-            key={index}
-            title={item.title}
-            index={index}
-            isActive={isActive(index)}
-            setActiveIndex={setActiveIndex}
-          >
-            {renderBody(item, index, items)}
-          </AccordianItem>
-        );
-      })}
-    </div>
-  );
-};
+}) => (
+  <div className="accordion">
+    {items.map((item, index) => (
+      <AccordianItem
+        key={index}
+        title={item.title}
+        index={index}
+        isActive={isActive(index)}
+        setActiveIndex={setActiveIndex}
+      >
+        {renderBody(item, index, items)}
+      </AccordianItem>
+    ))}
+  </div>
+);
 
 const BasicAccordian = withBasicAccordianHooks(AccordianContainer);
 const AlwaysOpenAccordian = withAlwaysOpenAccordianHooks(AccordianContainer);
 
 AccordianContainer.defaultProps = defaultProps;
 
-export { AlwaysOpenAccordian, BasicAccordian };
+export {
+  AlwaysOpenAccordian,
+  BasicAccordian,
+  BasicAccordianProps,
+  AccordianProps,
```

Comment:
do you rather want to call these types? or are these props? i am confused

Reply:
so these are propsTypes, should i rename them as AccordianType and BasicAccordianType?

Reply:
maybe, i guess that would make more sense